### PR TITLE
Copy center option from original RowLayout

### LINKED ISSUE #404: JFace RowLayoutFactory does not copy center option
The JFace RowLayoutFactory does not copy the center option in org.eclipse.jface.layout.RowLayoutFactory.copyLayout(RowLayout) from the original layout.

```
public static RowLayout copyLayout(RowLayout layout) {
		RowLayout result = new RowLayout(layout.type);
		result.marginBottom = layout.marginBottom;
		result.marginTop = layout.marginTop;
		result.marginLeft = layout.marginLeft;
		result.marginRight = layout.marginRight;
		result.marginHeight = layout.marginHeight;
		result.marginWidth = layout.marginWidth;

		result.fill = layout.fill;
		result.justify = layout.justify;
		result.pack = layout.pack;
		result.spacing = layout.spacing;
		result.wrap = layout.wrap;

		result.type = layout.type;

		return result;
	}
```

I have create a PR with the fix: #405

### PR DESCRIPTION
**Pull request by glatuske**
Add copy option to copyLayout
Remove type option (already set via constructor)

### CODE REVIEW

**Comment by laeubi** [comment]
@glatuske can you make sure you used the same e-mail for your commit that was used to sign the ECA and you have added your github account name to the profile?

**Comment by glatuske** [comment]
@laeubi I have re-commited with my e-mail I have signed ECA.

File: bundles/org.eclipse.jface/src/org/eclipse/jface/layout/RowLayoutFactory.java [hunk 1]
```diff
@@ -336,13 +336,12 @@ public static RowLayout copyLayout(RowLayout layout) {
 		result.marginWidth = layout.marginWidth;
 
 		result.fill = layout.fill;
+		result.center = layout.center;
 		result.justify = layout.justify;
 		result.pack = layout.pack;
 		result.spacing = layout.spacing;
 		result.wrap = layout.wrap;
 
-		result.type = layout.type;
```
Comment (laeubi) [inline bundles/org.eclipse.jface/src/org/eclipse/jface/layout/RowLayoutFactory.java:344]: Why do not copy the type anymore?

Comment (glatuske) [reply bundles/org.eclipse.jface/src/org/eclipse/jface/layout/RowLayoutFactory.java:344]: As mentioned in commit message: The has been already provided via constructor (line 330).

`RowLayout result = new RowLayout(layout.type);`

Comment (jukzi) [reply bundles/org.eclipse.jface/src/org/eclipse/jface/layout/RowLayoutFactory.java:344]: LGTM

**Pull request merged**